server/server.js

```diff
@@ -0,0 +1,122 @@
+const express = require('express')
+const bodyParser = require('body-parser');
+const cors = require('cors');
+
+require('dotenv').config();
+
+// imports the database connection
+const db = require('./db/db-connection.js');
+
+const app = express();
+const PORT = process.env.PORT || 3001;
+
+// middleware for parsing application/json
+app.use(cors());
+app.use(express.json());
+app.use(bodyParser.json());
+
+// test - creates an endpoint for the route  "/"
+app.get('/', (req, res) => {
+    res.json({ message: 'Hello from ExpressJS with React-Vite' });
+});
```

Comment:
It was used to test the connection of the server to the client, by passing the message as a data prop to be render on the webpage. It will be removed after this PR has been documented.